docs/js/controls-toggles.js

```diff
@@ -0,0 +1,12 @@
+export default 
+  window.requestAnimationFrame(() => {
+    let slider = document.querySelector("#slider1");
+    let counter = document.querySelector(".a-slider--value");
+
+    slider.addEventListener("input", function (){
```

Comment:
```suggestion
    slider.addEventListener("input", () => {
```

or just add spacing (maybe you should use a linter or something like that xD)

```suggestion
    slider.addEventListener("input", function () {
```

Reply:
d762264